p4-16/psa/PSA.mdk

```diff
@@ -70,7 +71,11 @@ on where the packets should go. After the ingress pipeline, the packet
 may be buffered and/or replicated (sent to multiple egress ports). For
 each such egress port, the packet passes through an egress match
 action pipeline and a checksum update calculation before it is
-deparsed and queued to leave the pipeline..
+deparsed and queued to leave the pipeline. Note: the checksum
+operations are available -- validation in the parser,
+and checksum computation and update in deparser (see also Table [])
```

Comment:
I don't know how to make a cross-reference to a table, but I do know that a cross-reference to a section can be done like so:
```
[#sec-extern-restrictions]
```
Is the table in that section the one you intended to cross-reference here?  If so, the section that table in is quite short, so a cross-reference to the whole section should be helpful enough to find the table.